jerry-core/ecma/operations/ecma-regexp-object.c

```diff
@@ -1693,8 +1693,7 @@ ecma_regexp_exec_helper (ecma_object_t *regexp_object_p, /**< RegExp object */
   uint32_t index = 0;
   if (bc_p->header.status_flags & (RE_FLAG_GLOBAL | RE_FLAG_STICKY))
   {
-    ecma_string_t *lastindex_str_p = ecma_get_magic_string (LIT_MAGIC_STRING_LASTINDEX_UL);
-    ecma_value_t lastindex_value = ecma_op_object_get_own_data_prop (regexp_object_p, lastindex_str_p);
+    ecma_value_t lastindex_value = ecma_op_object_get_by_magic_id (regexp_object_p, LIT_MAGIC_STRING_LASTINDEX_UL);
```

Comment:
Please add a note about that, this operation may throw error, but ToInteger handles it.